website/docs/docs/building-a-dbt-project/building-models/python-models.md

```diff
@@ -0,0 +1,378 @@
+---
+title: "Python models"
+---
+
+:::info Beta
+
+This feature is arriving in dbt Core v1.3, which is currently in beta. We encourage you to:
```

Comment:
is it worth highlighting that it's not available in cloud yet (unless I'm missing something)?

Reply:
Is it also worth making a callout that they're only available in BQ, Snowflake, and Databricks earlier here?

Reply:
> is it worth highlighting that it's not available in cloud yet (unless I'm missing something)?

It will be available in dbt Cloud, but with some functional gaps that we're working to close over the next ~month

> Is it also worth making a callout that they're only available in BQ, Snowflake, and Databricks earlier here?

Good call! Cody had the same thought and added a link